.pre-commit-config.yaml

```diff
@@ -55,6 +55,27 @@ repos:
       minimum_pre_commit_version: 2.8.0
       name: Check notebooks have watermark (see Jupyter style guide from PyMC docs)
       types: [jupyter]
+    - id: no-internal-links
+      name: Check no internal links are in the docs
+      description: >-
+        'A quick check to prevent urls pointing to pymc docs'
+      files: ^examples/
+      exclude: >
+          (?x)(conf.py)
+      entry: >
+          (?x)(arviz-devs.github.io|
+               aesara.readthedocs.io|
+               aeppl.readthedocs.io|
+               pymc-experimental.readthedocs.io|
+               docs.pymc.io|
+               numpy.org/doc|
+               pymc-examples.readthedocs.io|
+               myst-parser.readthedocs.io|
```

Comment:
The two myst ones can be removed, but `python.arviz.org` and `docs.xarray.dev` should be added

Reply:
Added in https://github.com/pymc-devs/pymc-examples/pull/311/commits/22e83a6e6c6f1b3f7e8ac70c767422268420b6af